### PR TITLE
Migrating to contracts 2.0 and marketplace api

### LINKED ISSUE #16: Add CLI documentation for all the commands

### LINKED ISSUE #38: The CLI needs to be updated to NVM 2.x

### LINKED ISSUE #41: The CLI needs provide general documentation and commands supported

### LINKED ISSUE #46: The CLI needs to have a command to deploy artifacts from the artifacts repo

### PR DESCRIPTION
**Pull request by aaitor**
## Description

Migration to NVM 2.0

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

File: .github/workflows/release-github.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: |
+            target/*.jar
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (r-marques) [inline .github/workflows/release-github.yml]: this needs to be adapted to something like this in order to work https://github.com/nevermined-io/sdk-py/blob/master/.github/workflows/release-github.yml

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -30,14 +39,19 @@ jobs:
...
           
           export KEEPER_OWNER_ROLE_ADDRESS="0xe2DD09d719Da89e5a3D0F2549c7E24566e947260"
-          export KEEPER_VERSION=v1.3.12
-          export GATEWAY_VERSION=v0.12.8
-          export METADATA_VERSION=v0.5.5
+          export KEEPER_VERSION=latest
+          export GATEWAY_VERSION=develop
+          export COMPUTE_API_VERSION=latest
+          export MARKETPLACE_API_VERSION=latest
+          export SUBGRAPH_VERSION=v0.3.0-rc4
```
Comment (r-marques) [inline .github/workflows/testing.yml]: rc5 is currently building

File: package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
     "ncli": "./dist/src/index.js"
   },
   "dependencies": {
-    "@nevermined-io/nevermined-sdk-js": "0.19.20",
+    "@nevermined-io/nevermined-sdk-js": "0.20.0-rc6",
```
Comment (r-marques) [inline package.json]: I need to make a new release with the new subgraph clients. it will probably be rc7

File: src/commands/accounts/list.ts [hunk 4]
```diff
@@ -17,19 +17,21 @@ import {
...
   const token = await loadToken(nvm, config, verbose)
 
   logger.debug(chalk.dim('Loading account/s ...'))
 
-  let accounts
-  if (account) accounts = [new Account(account)]
-  else accounts = await nvm.accounts.list()
+  // let accounts
+  // if (account) accounts = [account]
+  // else accounts = await nvm.accounts.list()
```
Comment (r-marques) [inline src/commands/accounts/list.ts]: should this be removed?

File: src/commands/agreements/listAgreements.ts [hunk 5]
```diff
@@ -23,7 +24,7 @@ export const listAgreements = async (
     logger.info(chalk.dim(`No agreements for '${chalk.whiteBright(ddo.id)}'!`))
   }
 
-  agreements.sort((a, b) => a.blockNumberUpdated - b.blockNumberUpdated)
+  // agreements.sort((a, b) => a.blockNumberUpdated - b.blockNumberUpdated)
```
Comment (r-marques) [inline src/commands/agreements/listAgreements.ts]: remove?

File: src/commands/nfts/showNft.ts [hunk 6]
```diff
@@ -37,9 +38,9 @@ export const showNft = async (
     )
   )
 
-  let userAccount
-  if (account) userAccount = new Account(account)
-  else [userAccount] = await nvm.accounts.list()
+  // let userAccount
+  // if (account) userAccount = new Account(account)
+  // else [userAccount] = await nvm.accounts.list()
```
Comment (r-marques) [inline src/commands/nfts/showNft.ts]: remove?

**Review by r-marques — APPROVED** [review]
lgtm

File: package.json (hunk 3 above)
Comment (aaitor) [reply package.json]: I think we can merge this one and raise a new PR as soon as this final `sdk-js` `v0.20.0` is released

**Pull request merged**